src/header/header.scss

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../user-icon/index';
+@import '../icon/index';
+
+
+@mixin iui-header {
+  @include iui-reset;
+  @include iui-font-family;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  height: $iui-baseline * 5;
+  transition: height $iui-speed-fast ease-out;
+  @include themed {
+    background-color: t(iui-color-background-1);
+    border-bottom: $iui-xxs solid t(iui-color-background-5);
+    color: t(iui-text-color);
+  }
+
+  > .iui-left,
+  > .iui-center,
+  > .iui-right {
+    display: flex;
+    align-items: center;
+    height: 100%;
+    width: 33%;
+  }
+
+  > .iui-left {
+    &:nth-last-child(2) {
+      width: unset;
+      overflow: hidden;
```

Comment:
> Unsure why only the top example is working correctly & has a blue stripe underneath the active button. Any ideas?
> 
> ![Screen Shot 2021-04-23 at 6 17 46 PM](https://user-images.githubusercontent.com/849817/115934984-11036300-a447-11eb-86c6-9f8d9e171445.png)

Its because of this `overflow: hidden`, the blue bar overflows the `iui-left` container, but is hidden in this case, so it does not show up, I tried playing with `overflow-x` and `overflow-y`, but whenever any is set to hidden, the box-shadow do not show up.

Reply:
Changing `width: unset; overflow: hidden;` to simply `flex-basis: 100%` gives the expected left side shrinking behavior without hiding the blue bar below, I cant see any down-side, maybe that is our solution here.